chronos/transports/stream.nim

```diff
@@ -50,7 +50,12 @@ type
     # get stuck on transport `close()`.
     # Please use this flag only if you are making both client and server in
     # the same thread.
-    TcpNoDelay
+    TcpNoDelay # deprecated: Use ClientFlags.TcpNoDelay
+
+  ClientFlags* {.pure.} = enum
```

Comment:
The only name which greatly explains all the flags inside is `SocketFlags`, because all the flags are related to sockets only.

Reply:
Done